Sources/MongoSwift/BSON/BsonEncoder.swift

```diff
@@ -507,16 +509,24 @@ private class MutableArray: BsonValue {
 
     var count: Int { return array.count }
 
+    func insert(_ value: BsonValue?, at index: Int) {
```

Comment:
I realize the method isn't new (just relocated in the diff) and the class isn't implementing a protocol, but is this a method related to some Swift array protocol?